src/main/resources/messages/LibertyBundles.properties

```diff
@@ -104,15 +104,20 @@ gradle.test.report.does.not.exist=Gradle Test Report Does Not Exist
 test.summary=Test Summary
 
 # View unit test report action
-view.unit.test.report=view unit test report
+view.unit.test.report=View unit test report
```

Comment:
I think this one is supposed to stay lowercase since we have the uppercase covered below.